packages/serverless-helpers/src/helpers/testFunctionNames.ts

```diff
@@ -0,0 +1,39 @@
+import { AWS } from '@serverless/typescript';
+
+const MAX_AWS_LAMBDA_NAME_LENGTH = 64;
+const STAGE = 'production';
```

Comment:
are we always testing with `production` as stage? 
this is considering people always use our convention of `dev`, `staging`, `production`

Reply:
Hmm that is a good point. However maybe we should open a separate issue on this, what do you think?

Reply:
This test is here to cover the worst case scenario (with the longest name), so we use `production` as it is the one with the most letters

Reply:
What we can do is add a comment to explain that in the file, WDYT ?

Reply:
I added it as a jsdoc so it also available in service tests